src/scala/META-INF/native-image/org/pantsbuild/zinc/compiler/native-image.properties

```diff
@@ -0,0 +1,12 @@
+ImageName = zinc-pants-native
+Args = -H:SubstitutionResources=${.}/substitutions.json \
+       -H:ResourceConfigurationResources=${.}/resource-config.json \
+       -H:ReflectionConfigurationResources=${.}/reflect-config.json \
+       --enable-all-security-services --allow-incomplete-classpath \
+       --report-unsupported-elements-at-runtime \
+       --delay-class-initialization-to-runtime=scala.tools.nsc.interpreter.IMain \
```

Comment:
Can you add a comment explaining how this list was worked out, and when it should be modified in the future?

Reply:
Done in the README.md in the same directory!